src/stream/src/executor/merge.rs

```diff
@@ -35,6 +35,9 @@ pub struct MergeExecutor {
     /// Belonged actor id.
     actor_id: ActorId,
 
+    /// Actor id as String
+    actor_id_str: String,
```

Comment:
No need to put it in `self`? 🤔

Reply:
The `execute_inner` is also only be called once, so it's okay to make it a local variable. :)

Reply:
Ah, you are right!